### PR TITLE
CHECK-41-patch-objects

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,38 +47,59 @@ Additional checks for FASTQ file:
...
-    -e AWS_ACCESS_KEY_ID=xxxxxxxx -e AWS_SECRET_ACCESS_KEY=xxxxxxxx\
-    -e ENCODE_ACCESS_KEY=xxxxxxxx -e ENCODE_SECRET_KEY=xxxxxxxx\
-    checkfiles pytest
-```
+In order to deploy a checkfiles EC2 instance (right now it makes most sense to run on igvf-staging account, to easily have access both to sandbox and prod from same machine).
+1. Log into igvf-staging aws console.
+2. Go to EC2 console and in the left tab select `AMIs`
+3. Select the newest `checkfiles__ami` (`ami-02001e7412122509a`)
+4. Click `Launch instance from AMI`.
+5. Select instsance type, and add storage. 200GB should be enough for anything.
```
Comment (mingjiecn) [inline README.md:81]: can you recommend the instance type here?

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Need to refactor the code to use fetch_etag_for_uuid()

File: README.md (hunk 1 above)
Comment (ottojolanki) [reply README.md:81]: Yea, I will add the recommendation to use the t2 instances.

File: src/checkfiles/checkfiles.py [hunk 2]
```diff
@@ -347,10 +347,30 @@ def fetch_pending_files_metadata(portal_uri: str, portal_auth: PortalAuth) -> li
     return metadata
 
 
+def fetch_etag_for_uuid(portal_uri: str, file_uuid: str, portal_auth: PortalAuth) -> str:
```
[the remark this answers is not in the document]
Comment (ottojolanki) [reply src/checkfiles/checkfiles.py:350]: good catch. fixing.

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**